### ISSUE TITLE
Elasticsearch is not on hackage :)

### ISSUE THREAD

**Issue by mwotton** [ocharles/Elasticsearch#4]
what needs to be done before making a release?

**Comment by ocharles** [comment]
Nothing needs to be done really - if you think this is in a state to
release then I can do that. Should we claim the "elasticsearch" package
name?

**Comment by mwotton** [comment]
I reckon so, no-one else is using it and there isn't a natural other name - it's not a conduit or pipes thing. It's obviously not complete, but it's useful already, and hopefully it'll at least save other people redoing the basic stuff.

**Comment by ocharles** [comment]
Alright, had a look at the code just now and it's not quite at a Hackage ready state. The code itself is fine, but I want to do a little bit of house keeping first. Fingers crossed, I'll find time to do this tomorrow.

**Comment by mwotton** [comment]
Yeah, I have some more changes in the wings too, perhaps we should hold off.

Basically, I'm running some load testing and elasticsearch seems to just
drop acknowledged requests. If I push up my code to detect it, reckon you
could sanity check it?

On Thu, Jan 9, 2014 at 11:04 AM, Oliver Charles notifications@github.comwrote:

> Alright, had a look at the code just now and it's not quite at a Hackage
> ready state. The code itself is fine, but I want to do a little bit of
> house keeping first. Fingers crossed, I'll find time to do this tomorrow.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ocharles/Elasticsearch/issues/4#issuecomment-31890188
> .

## 

A UNIX signature isn't a return address, it's the ASCII equivalent of a
black velvet clown painting. It's a rectangle of carets surrounding a
quote from a literary giant of weeniedom like Heinlein or Dr. Who.
        -- Chris Maeda

**Comment by ocharles** [comment]
Sure, just make sure I get a notification and I'll check it out.

**Comment by mwotton** [comment]
ok, sent.

Think we probably also need a better story for failure and retry on multiple servers, as they'll be API breaking changes.

**Comment by ocharles** [comment]
Closing in favor of [`bloodhound`](http://hackage.haskell.org/package/bloodhound). I am no longer working on this library (or doing anything with ElasticSearch), so it's unlikely to see the light of day.
